### DISCUSSION TITLE
CSS is not working in Github.io

### DISCUSSION THREAD

**Discussion by jasminehuelgas** [community/community#143525] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

https://jasminehuelgas.github.io/
https://github.com/jasminehuelgas/jasminehuelgas.github.io.git

Here's the website that I created using Github pages. The website is working perfectly fine on my local machine. However, when I uploaded it to Github, it is not working in github.io. Can anyone help me fix this?

**Comment by Aiche-H** [comment]
I’ve been trying to access the issue page you mentioned, but I’m not seeing any CSS issues on my end. Every page seems to be loading correctly with styles applied.

Could you please provide more specific details about the issue, such as:

* **The exact URL** of the page where the CSS is not working
* **The specific browser and device** you’re using
* **A screenshot** of the page as it appears on your end

This information will help me better understand and troubleshoot the problem.

**Reply by jasminehuelgas** [reply]
Hi, thank you for your help! I fixed the issue now, it seems that my website is now working fine.

**Comment by davevad93** [comment]
Hi @jasminehuelgas ,

I've checked your portoflio and the issue is within your `css`, exactly in the first line of the `body` tag:

```css
body{
	background: url("../img/back.JPG");
        ...
}
```

GitHub pages is case sensitive, change it to `/img/back.jpg`.

Result:

![11](https://github.com/user-attachments/assets/14e6f61a-66b2-4b63-bc66-9c5810f1346c)

**Reply by jasminehuelgas** [reply, marked as answer]
Hi, thank you for your help! I fixed the issue now, it seems that my website is now working fine.

**Reply by davevad93** [reply]
You're welcome @jasminehuelgas , I'm glad you got it sorted out, remember that GitHub Pages is case sensitive so for instance if a file is named `image.jpg` or `home.html` you must reference it exactly like that in your code.
